libs/galley-types/test/unit/Test/Galley/Permissions.hs

```diff
@@ -27,7 +27,7 @@ import Wire.API.Team.Role
 tests :: TestTree
 tests =
   testGroup
-    "permsToInt"
+    "Role, permsToInt"
```

Comment:
i added this because i spent too much time searching for this test by key 'Role'  :)

Reply:
```suggestion
    "permsToInt for rolePermissions"
```

How about that? It's a bit closer to what actually happens here. However, it's not a mandatory/importang change. (I could live without it. So, please feel free to ignore :smile_cat: )

Reply:
```suggestion
    "permsToInt / rolePermissions / serialization of `Role`s"
```